apps/web/src/components/Dropdown/Dropdown.tsx

```diff
@@ -34,40 +46,20 @@ export const Dropdown: React.FC<DropdownProps> = function ({
               aria-hidden="true"
             />
           </span>
-        </Listbox.Button>
+        </ListboxButton>
         <Transition
           as={Fragment}
```

Comment:
This is the default value, so we do not have to set it:
![image](https://github.com/user-attachments/assets/2eb3c9df-16d1-4989-a703-ebd46eea6d14)

https://headlessui.com/react/transition

Reply:
Removed :+1: